src/Utils/root.css

```diff
@@ -32,4 +32,5 @@ body.active {
 
 .btn-success{
 	background-color: var(--primary);
+	color:white;
```

Comment:
Use colour which are predeclared in root folder

Reply:
will do that